src/main/java/com/google/starfish/servlets/HandleNotesServlet.java

```diff
@@ -53,6 +53,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       response.setStatus(HttpServletResponse.SC_NOT_ACCEPTABLE);
       return;
     }
+    
+    blobKey = "/serve-notes?key=" + blobKey;
```

Comment:
Since this is no longer a blobKey, but the sourceUrl, could we rename it that and trickle down the change?

Reply:
Yep I'll make that change!